tox.ini

```diff
@@ -1,32 +1,33 @@
 [tox]
 envlist =
+    py{3.10,3.11,3.12,3.13}-django{5.1}-psycopg{2,3}
     py{3.10,3.11,3.12}-django{5.0}-psycopg{2,3}
     py{3.8,3.9,3.10,3.11,3.12}-django{4.2}-psycopg{2,3}
 
 [testenv]
 usedevelop = True
 allowlist_externals = bash
 commands =
-    py{3.12}-django{5.0}-psycopg{3}: flake8
-    py{3.12}-django{5.0}-psycopg{3}: isort . --check --diff
```

Comment:
should `py{3.13}-django{5.1}-psycopg{3}` used instead `3.12` , this should fix tests

Reply:
Of course 🤦🏻